lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +81,40 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if isinstance(unknown_profile, dict) == False or isinstance(profile_to_compare, dict) == False:
+        return None
+    if 'name' not in unknown_profile or  'freq' not in unknown_profile:
+        return None
+    if 'name' not in profile_to_compare or 'freq' not in profile_to_compare:
+        return None
+
+    sorted_profile_to_compare = dict(sorted(profile_to_compare.items()))
+    sorted_unknown_profile = dict(sorted(unknown_profile.items()))
+
+    predicted_tokens = sorted_profile_to_compare.get('freq')
+    actual_tokens = sorted_unknown_profile.get('freq')
+
+    for key in predicted_tokens.keys():
+        if key not in actual_tokens:
+            actual_tokens[key] = 0
+
+    for key in actual_tokens.keys():
+        if key not in predicted_tokens:
+            predicted_tokens[key] = 0
+
+    predicted = []
+    for value in predicted_tokens.values():
+        predicted.append(value)
+
+    actual = []
+    for value in actual_tokens.values():
+        actual.append(value)
+
+    mse = calculate_mse(predicted, actual)
+    mse = round(mse, 3)
```

Comment:
Is this the right place for round here?

Reply:
I've put this line in the body of calculate_mse function, fixed

Reply:
I can't see where in the description of MSE function information about the necessity to round the output value. It is expected later in the tasks for mark 10, but at this point, values are to be returned as is. Correct me if I'm wrong.

Reply:
I thought that some tests wouldn't run if I don't round MSE, but all of them do, thank you, fixed